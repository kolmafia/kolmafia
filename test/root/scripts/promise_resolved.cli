js new Promise((resolve) => resolve("foo"))
