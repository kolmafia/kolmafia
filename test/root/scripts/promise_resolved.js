const {print} = require("kolmafia");
new Promise((resolve) => resolve("foo")).then((value) => print(value));
