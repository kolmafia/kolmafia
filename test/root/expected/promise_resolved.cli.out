Returned: org.mozilla.javascript.NativePromise@37c1e2a4